pkg/mock/mockcluster/mockcluster.go

```diff
@@ -82,12 +82,12 @@ func (mc *Cluster) IsRegionHot(region *core.RegionInfo) bool {
 }
 
 // RegionReadStats returns hot region's read stats.
-func (mc *Cluster) RegionReadStats() map[uint64][]*statistics.HotSpotPeerStat {
+func (mc *Cluster) RegionReadStats() map[uint64][]*statistics.HotPeerStat {
 	return mc.HotSpotCache.RegionStats(statistics.ReadFlow)
```

Comment:
It seems that most places replace "hotSpot" with "hot". Needs to be changed here?